packages/mgt-components/src/components/mgt-agenda/mgt-agenda.ts

```diff
@@ -265,7 +268,7 @@ export class MgtAgenda extends MgtTemplatedComponent {
 
     // Render list
     return html`
-      <div dir=${this.direction} class="agenda${this._isNarrow ? ' narrow' : ''}${this.groupByDay ? ' grouped' : ''}">
+      <div dir=${this.direction} class="agenda${this.groupByDay ? ' grouped' : ''}">
```

Comment:
_nit:_ perhaps a classMap would be more readable here and in other places where the narrow class is being interpolated in?